specifications/api/swagger.yaml

```diff
@@ -2195,6 +2335,17 @@ x-responsesPostExternalTransaction: &responsesPostExternalTransaction
       application/json:
         schema: *ApiTxId
 
+x-responsesPostAnyAddress: &responsesPostAnyAddress
+  <<: *responsesErr400
+  <<: *responsesErr405
```

Comment:
I think it's ok to drop `405` now to be consistent with other endpoints after recent error codes cleanup/documentation. (cf: https://github.com/input-output-hk/cardano-wallet/pull/2258#discussion_r515004888)

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/2253/commits/595999169a24d1637760f2f14f02c4af9d3cc4ad